nodes/Raise.py

```diff
@@ -1 +1,47 @@
-raise ZeroDivisionError()
+"""
+Raise astroid node
+
+This node represents the "raise" statement in Python, which is used to raise
+exceptions. This statement also handles exception chaining using the "from"
+clause, which is included in the Raise node.
+
+Attributes:
+    - exc    (node)
```

Comment:
Make sure to capitalize `Node` when referring to it as a class name. Also, express the fact that this attribute could be none in its type: `Node | None`